assets/js/components/Health/HealthIcon.jsx

```diff
@@ -11,13 +11,22 @@ import {
 import Spinner from '@components/Spinner';
 import classNames from 'classnames';
 
-const HealthIcon = ({ health = undefined, centered = false }) => {
+const HealthIcon = ({
+  health = undefined,
+  centered = false,
+  hoverOpacity = true,
+}) => {
+  const hoverOpacityClass = {
+    'hover:opacity-75': hoverOpacity,
+    'hover:opacity-100': !hoverOpacity,
+  };
```

Comment:
Your call but keep in mind that you can also just pass `className` down to the rendered component